### ISSUE TITLE
Defective lymphocyte proliferation to anti-CD3

### ISSUE THREAD

**Issue by jpazmandi** [obophenotype/human-phenotype-ontology#6120]
**Preferred term label:**
Defective lymphocyte proliferation to anti-CD3

**Synonyms**

**Definition (free text, please give PubMed ID)**
Defective proliferation of lymphocyte following anti-CD3 stimulation.

**Parent term (use hpo.jax.org/app)**
HP:0031379

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-3411-9598

**Comment by pnrobinson** [comment]
This is synonymous with
https://hpo.jax.org/app/browse/term/HP:0031381
I will add the label as a synonym
